src/pages/patientView/trialMatch/TrialMatchTableUtils.ts

```diff
@@ -0,0 +1,129 @@
+import {
+    IArmMatch, IClinicalGroupMatch, IDetailedTrialMatch, IGenomicGroupMatch, IGenomicMatch, ITrial,
+    ITrialMatch, IArm, IDrug
+} from "../../../shared/model/MatchMiner";
+import * as _ from 'lodash';
+
+export function groupTrialMatchesById(trials: ITrial[], trialMatches:ITrialMatch[]): IDetailedTrialMatch[] {
+    const matchesGroupedById = _.groupBy(trialMatches, (trial: ITrialMatch) => trial.id);
+    let matchedTrials: IDetailedTrialMatch[] = [];
+    _.forEach(matchesGroupedById, (trialGroup, trialId) => {
```

Comment:
you could probably do matchedTrials = _.map(matchesGroupedById).  it's better style to use direct assignment.  no big deal

Reply:
updated